drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/base/CheckFactory.java

```diff
@@ -67,8 +66,6 @@ public CheckFactory( final AnalyzerConfiguration configuration ) {
                                                                        configuration  ),
                                       new DetectDeficientRowsCheck( ruleInspector,
                                                      configuration ),
-                                      new RangeCheck( ruleInspector,
```

Comment:
Hi @mariofusco just question, why the new `SingleRangeCheck` is not registered on this place but in the `RuleInspectorCache`?